src/containers/ViewArticle/ViewArticle.js

```diff
@@ -45,6 +47,35 @@ class ViewArticle extends Component {
     this.props.rateArticle(this.props.match.params.articleslug, newRating);
   }
 
+  /**
+   *
+   * @returns {func} - bookmark
+   * @memberof ViewArticle
+   */
+  bookmark() {
+    const bookmarks = this.state.bookmarks
```

Comment:
you can destructure instead of assigning

Reply:
Destructuring will not work here because I am not assigning bookmarks to this.state.bookmarks. 
Instead bookmarks here is the result of the filter on this.state.bookmarks